fs/memoryfs.py

```diff
@@ -444,7 +444,7 @@ def makedir(
                 parent_dir.set_entry(dir_name, new_dir)
             return self.opendir(path)
 
-    def move(self, src_path, dst_path, overwrite=False):
+    def move(self, src_path, dst_path, overwrite=False, preserve_time=False):
```

Comment:
`MemoryFS` actually supports created/modified times, so you should be able to get it to preserve times here too.